websites/C/CTFtime/presence.ts

```diff
@@ -0,0 +1,243 @@
+type Functionlize<T> = {
+	[P in keyof T]: () => T[P];
+};
+
+interface Route extends Functionlize<Partial<PresenceData>> {
+	path: RegExp;
+}
+
+const enum Assets {
+	Logo = "https://i.imgur.com/ZHkLk9I.png",
+}
+
+enum Settings {
+	TIMESTAMP = "timestamp",
+	BUTTONS = "buttons",
+	COVERS = "images",
+}
+const { href, hostname } = document.location,
+	presence = new Presence({
+		clientId: "1253040451624370300",
+	}),
```

Comment:
```suggestion
	}),
	imageElement = `https://ctftime.org/${document.querySelector<htmlImageElement>("span2").querySelector("img").src}`;
```

Reply:
i don't see a point of doing this + it doesn't work

Reply:
nvm done